### DISCUSSION TITLE
Insufficient graph - plant pt assembly

### DISCUSSION THREAD

**Discussion by kfharris** [Kinggerm/GetOrganelle#260] [Q&A]
Hello @Kinggerm , 

I am currently working on a project involving the assembly of 56 plant plastomes. So far, 46 have been successfully assembled using the suggested base code (get_organelle_from_reads.py -1 input1.fastq.gz -2 input2.fastq.gz -o pt_output -R 15 -k 21,45,65,85,105 -F embplant_pt) or one of the several suggested variations within your FAQ page (-kmer increased and condensed, -w down by 10, using seed database, --reduce reads). However, I am currently running into two main problems with assembling the last 10. 

For boer_3325 and others like it, the GO run produced 3 scaffolds for the suggested base code. I then ran the same fastq files with an adjusted kmer (getorganelle/bin/get_organelle_from_reads.py -1 3325_S54_L002_R1_001.fastq.gz -2 3325_S54_L002_R2_001.fastq.gz -o boer_3325-GO2 -R 15 -k 21,55,85,115 -F embplant_pt), a seed database (-s lyciumcp.fasta ), a reduced read coverage (--reduce-reads-for-coverage 15000000), and a reduced word size (-w 178), all of which produced the same result (insufficient graph and 3 scaffolds). When aligning the scaffolds to an existing plastome, it is clear that one is the IR section and another is the SSC. However, it almost appears as though the LSC is reversed or reverse complemented, and will not assemble. The scaffolds have all been blasted to GenBank and the results there also seem as though these are the identities of the scaffolds. I can't seem to find additional information either in the 2020 paper or on the FAQ page. I have attached the log file for the first run of this example, as well as the csv file of the output. I was just wondering if you had any ideas for how to proceed with this assembly issue? The LSC inversion issue seems to be happening to a few of the failing plastomes. If other files or information would be helpful, please let me know and I'd be happy to share them. Thank you! 

Kathleen
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/11276529/get_org.log.txt)
[extended_K85.assembly_graph.fastg.extend-embplant_pt-embplant_mt.csv](https://github.com/Kinggerm/GetOrganelle/files/11276533/extended_K85.assembly_graph.fastg.extend-embplant_pt-embplant_mt.csv)

**Comment by Kinggerm** [comment]
I did not understand what you referred to as the LSC inversion issue. 

The first thing to improve the assembly is whether it is organelle-sufficient (see [FAQ](https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-should-i-do-with-incomplete-resultbroken-assembly-graph)). One should use a visual check of the assembly graph using Bandage. Please confirm.
